### DISCUSSION TITLE
Time difference between VM and Docker logs causing lag ??

### DISCUSSION THREAD

**Discussion by lymanaukuleles** [jertel/elastalert2#833] [Q&A] [answered]
I have been experiencing lag between event in Kibana and alert in email and have been trying to find the culprit behind this. I noticed the time between the VM and the docker logs are different. This is the current time on the VM using the `date` command:
`Mon May  2 10:09:19 HST 2022`

And this is the latest time in the docker logs:
`2022-05-02 20:08:57`

Just wondering if this could be the reason between the event being indexed into Elasticsearch versus the alert being delivered to my email.

**Comment by jertel** [comment, marked as answer]
It's common practice for servers to run in UTC time. So the 20:08 time on the docker logs is expected. Check your Elastic data and see if the @timestamps are also in UTC time. If they are not in UTC time, I recommend adjusting your data ingest pipelines so that data pushed into ES is always timestamped in UTC time. This removes the complexities of timezone and daylight savings, which are error prone by system administrators and software developers.
